cmd/kube-scheduler/app/options/options_test.go

```diff
@@ -64,7 +65,7 @@ func TestSchedulerOptions(t *testing.T) {
 	configFile := filepath.Join(tmpDir, "scheduler.yaml")
 	configKubeconfig := filepath.Join(tmpDir, "config.kubeconfig")
 	if err := ioutil.WriteFile(configFile, []byte(fmt.Sprintf(`
-apiVersion: componentconfig/v1alpha1
+apiVersion: kubescheduler.config.k8s.io/v1alpha1
```

Comment:
leave a test case in place that attempts to load a `componentconfig.v1alpha1` file and ensures it errors with a good message

Reply:
Is this done?

Reply:
yes, the `oldconfigFile` case below